<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamesettingdialog</class>
 <widget class="QDialog" name="gamesettingdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>401</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>100</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>보드 크기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>100</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>장애물 개수</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="maximum">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>100</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>선공 플레이어</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>흑</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>백</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>무작위</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>게임 설정</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>660</y>
     <width>193</width>
     <height>29</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxTimeLimit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>제한 시간 모드 사용</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBaseTime">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>610</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>600</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxIncrement">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>610</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>610</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>총 시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>610</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>증가 시간</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
